test.py

```diff
@@ -0,0 +1,17 @@
+import random
+import pandas as pd
+import numpy as np
+
+
+def test(cols_size, rows_size):
```

Comment:
```suggestion
def test(cols_size: int, rows_size: int) -> None:
```
Here is another way to somehow describe the function parameters and return value/type. "`Annotations`" are really useful for programmers and for code inspection by IDE as they somehow can replace type checking as you may know from `C` or `C++`. Ofc. it is not the same and intentions of introducing such annotations were not aimed for that purpose. We can treat them as additional help during someone code reading. You can read more about them on the Internet.